<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="testcases.SearchJobsFunctionality" tests="1" time="55.775" ignored="0" failures="1" timestamp="10 Oct 2022 10:56:02 GMT" skipped="0" hostname="DESKTOP-1BE3706" errors="0">
  <testcase name="validSearchJobsFunctionality" time="55.775" classname="testcases.SearchJobsFunctionality">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Enter value =&gt; Tester is not contain with Searched value =&gt; COBOL Developer expected [true] but found [false],
	Enter value =&gt; Tester is not contain with Searched value =&gt; React Staff Software Developer expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Enter value => Tester is not contain with Searched value => COBOL Developer expected [true] but found [false],
	Enter value => Tester is not contain with Searched value => React Staff Software Developer expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at pageObjects.JobSearchPage.validateSearchFunctionality(JobSearchPage.java:66)
	at testcases.SearchJobsFunctionality.validSearchJobsFunctionality(SearchJobsFunctionality.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- validSearchJobsFunctionality -->
</testsuite> <!-- testcases.SearchJobsFunctionality -->
